terraform/oke/schema.yaml

```diff
@@ -17,69 +18,58 @@ source:
 locale: "en"
 
 variableGroups:
-  - title: "configuration inputs"
+  - title: "hidden inputs"
     variables:
       - tenancy_ocid
       - region
       - user_ocid
       - private_key_path
       - fingerprint
-      - generate_helm_template
-      - enable_dashboard_module
-      - enable_helm_module
       - boat_auth
       - boat_tenancy_ocid
+      - dev_switch_generate_helm_template
+      - dev_switch_install_helm
+      - dev_switch_use_local_helm_chart
+      - dev_switch_dashboards_module
+      - dev_switch_helm_module
+      - dev_switch_mgmt_agent_module
+      - dev_switch_logan_module
+      - dev_switch_livelab_module
+      - dev_switch_iam_module
       - compartment_ocid
       - logan_container_image_url
       - mgmt_agent_container_image_url
       - kubernetes_namespace
       - current_user_ocid
       - livelab_switch
+      - triggered_by_add_data_flow
+      - oke_cluster_name
+      - oke_cluster_entity_ocid
+      - stack_deployment_option
     visible: false
 
-  - title: "Select an OKE Cluster deployed in this region to start monitoring"
-    description: "Use CLI (Helm) if your cluster does not have public API Endpoint or restricted from accessing container-registry.oracle.com. See: https://github.com/oracle-quickstart/oci-kubernetes-monitoring"
+  - title: Select an OKE cluster deployed in this region to start monitoring.
+    description: "Use CLI (Helm) if your cluster does not have a public API endpoint or if it's restricted from accessing container-registry.oracle.com. See: https://github.com/oracle-quickstart/oci-kubernetes-monitoring"
```

Comment:
Rephrase to say:
Perquisites: OKE Cluster must have a public API endpoint and can pull image from container-registry.oracle.com. See https://github.com/oracle-quickstart/oci-kubernetes-monitoring for details.